web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1192,6 +1192,14 @@ export const formatters: Formatters = {
         ? `User [${user.user}] has spent a device enroll token`
         : `User [${user.user}] has failed to spend a device enroll token`,
   },
+  [eventCodes.DEVICE_UPDATE]: {
+    type: 'device',
+    desc: 'Device Update',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has updated a device`
```

Comment:
Would it be better to use "trusted device" as opposed to just "device" to make sure no one might confuse and think it's referring to an MFA device? 

If so, the other device trust events should be updated too.

Reply:
I think we do say "MFA" whenever we mean MFA, so that should help. There is also some nuance on whether every device registered is trusted, so I'm trying to avoid saying "trusted" for devices that aren't fully enrolled.

I can see it going either way, but leaning to leave as-is for now.